bulk_customer_import.py

```diff
@@ -65,18 +61,26 @@ def parse_csv(filename):
     return output
 
 
-def get_session(base_url, auth_user, auth_pass):
+def get_session(base_url, auth_user, auth_api):
     # init session
     logging.info("Initializing session")
     session = requests.Session()
-    session.auth = (auth_user, auth_pass)
-    url = base_url + "/rest/auth/1/session"
-    response = session.get(url)
+    session.auth = (auth_user, auth_api)
+    user_pass_concat = f"{auth_user}:{auth_api}"
+    user_pass_concat_bytes = user_pass_concat.encode()
+    user_pass_concat_base64 = base64.b64encode(user_pass_concat_bytes)
+    user_pass_concat_base64_str = user_pass_concat_base64.decode("UTF-8")
```

Comment:
I'm not sure that it's necessary to build the Basic auth header manually - IIRC think the 'Requests' http library builds the Basic auth hash header from the `auth_user` and `auth_pass` fields supplied - i.e. to authenticate with an API token instead of a password - just pass the api token for the `auth_pass` argument.

If anything, the reason why this script is failing to authenticate is because it's using cookie based authentication (via `jira_session`) which was deprecated in 2017.

Reply:
Don't worry about making any change to this though.

Reply:
Confirmed - its not necessary to build Basic hash - just pass api token as `auth_pass`